qatrfm/cli.py

```diff
@@ -87,6 +89,35 @@ def find_testcases(opt_test: Path):
     return load_testcases(basedir, files)
 
 
+def get_network_octet():
+    """
+    Find a non-used network in the system
+
+    To allow multiple environments co-exist, network ranges can't be
+    hardcoded. Otherwise, new libvirt virtual networks can't be created.
+    The default environment will create a network with range 10.X.0.0/24,
+    where X will be calculated dynamically according to the existing
+    networks on the system, starting from X=0, this offers 255 possible
+    isolated environments running at the same time.
+    """
+    locks_dir = '/tmp/qatrfm'
+    os.makedirs(locks_dir, exist_ok=True)
+    x = 0
+    while x < 255:
+        try:
+            f = open('{}/{}'.format(locks_dir, x), 'a')
```

Comment:
I need to keep the file object in this flow, otherwise garbage collector will delete the object and therefore the lock.